.github/workflows/build.yml

```diff
@@ -74,8 +74,8 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: [3.9]
-        numpy-version: [1.22]
+        python-version: [3.11]
+        numpy-version: [1.25]
```

Comment:
What are these versions for?

Reply:
penultimate version. Just being safe as in my experience GPU stuff is temperamental on latest versions.